.vscode/settings.json

```diff
@@ -6,11 +6,11 @@
     "out": true
   },
   "[typescript]": {
-    "editor.defaultFormatter": "vscode.typescript-language-features",
     "editor.codeActionsOnSave": {
       "quickfix.biome": "explicit",
       "source.organizeImports.biome": "explicit"
     }
   },
+  "editor.defaultFormatter": "biomejs.biome",
```

Comment:
Otherwise, Prettier formats the YAML with double quotes. This fixes that. It also just fixes TS formatting in general.